content/tutorials/custom-zk-chain/10.index.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Getting Started with ZK Chains
+description: Create and run your first ZK chain.
+---
+
+This tutorial shows you how to use the `zk_inception` CLI to run an Elastic Chain ecosystem and custom ZK chain locally:
+
+- You'll set up a local **Elastic Chain ecosystem**
+- You'll create a standard **ZK chain** and deploy a smart contract to it
+- You'll create a second ZK chain that uses a **custom  ERC20 base token**
+
+## Prerequisites
+
+- Make sure your machine satisfies the [system
+  requirements](https://github.com/matter-labs/era-compiler-solidity/tree/main#system-requirements).
+- If you aren't already familiar with deploying smart contracts on ZKsync Era, please  refer to the first section of the [quickstart tutorial](https://docs.zksync.io/build/start-coding/quick-start/deploy-your-first-contract).
+- For background on the Elastic Chain or ZK chains, read the [ZK Chains](https://docs.zksync.io/zk-stack/concepts/zk-chains) section in our docs.
```

Comment:
I know the page linked has the title as "ZK Chains" with C-capital, but I'll be going through and updating the titles to better match the guidelines we follow.

```suggestion
- For background on the Elastic Chain or ZK chains, read the [ZK chains](https://docs.zksync.io/zk-stack/concepts/zk-chains) section in our docs.
```